src/bakery_manager_mobile/lib/pages/home_page.dart

```diff
@@ -53,58 +53,25 @@ class _MyHomePageState extends State<MyHomePage> {
     }
   }
 
-  void _logout() async {
-    if (await ApiService.logout()) {
-      if (!mounted) {
-        return;
-      }
-      SessionManager().clearSession();
-      Navigator.pushReplacementNamed(context, loginPageRoute);
-    }
-  }
-
   @override
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(
         centerTitle: true,
         backgroundColor: const Color.fromARGB(255, 209, 125, 51),
-        shape: const RoundedRectangleBorder(
```

Comment:
Appbar makes a arrow in the top left add automaticallyImplyLeading: false, under centerTitle to remove this